integration-tests/src/test/kotlin/com/google/devtools/ksp/test/PlaygroundIT.kt

```diff
@@ -379,6 +379,17 @@ class PlaygroundIT(val useKSP2: Boolean) {
         project.restore(buildFile.path)
     }
 
+    @Test
+    fun testModuleName() {
+        File(project.root, "workload/build.gradle.kts")
+        val gradleRunner = GradleRunner.create().withProjectDir(project.root)
+        gradleRunner.withArguments("build").build().let { result ->
+            // TODO figure out why K2 uses the source set name and K1 uses the project name
+            val expectedName = if (useKSP2) "workload" else "main"
```

Comment:
According to the comment, should this be in the reversed order?

Reply:
Yes!